charts/fission-all/values.yaml

```diff
@@ -143,6 +155,18 @@ executor:
   ##
   resources: {}
 
+  ## Security Context
+  ## It holds pod-level and container level security configuration.
+  ## This section is experimental, and enabling it may introduce new challenges.
+  ## Ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1
+  securityContext:
```

Comment:
fetcher security context is used bit differently in Fission. It is used by buildermgr and executor. Please see above comments. 
We would need to think how to utilize fetcher securitycontext defined here.
We may need changes in Go code to utilize below defined values.